src/debug/helper.jl

```diff
@@ -83,8 +83,6 @@ function dbg_generate(
   )
 end
 
-dbg_generate(_data = Dict(); kwargs...) = dbg_generate(rand_pkg_name(), _data; kwargs...)
```

Comment:
Maybe it's enough to define

```julia
dbg_generate(_data; kwargs...) = dbg_generate(rand_pkg_name(), _data; kwargs...)
```

Can you try this, please?

Reply:
This doesn't work either because of lacking type annotation of `_data`. I've found that proper type annotation can avoid this ambiguity, see the new commits.